### PR TITLE
add colpali

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @osanseviero

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -130,6 +130,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"asset/GPT.pt"`,
 	},
+	"colpali": {
+		prettyLabel: "colpali",
+		repoName: "colpali",
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:135]:
```suggestion
		prettyLabel: "ColPali",
		repoName: "ColPali",
```

**Review by Wauplin — APPROVED** [review]
LGTM :)
Better to update https://huggingface.co/vidore/colpali/blob/main/README.md adding `library_name: colpali` before merging this PR so that https://huggingface.co/models?other=colpali is not empty.

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -130,6 +130,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"asset/GPT.pt"`,
 	},
+	"colpali": {
+		prettyLabel: "ColPali",
+		repoName: "ColPali",
+		repoUrl: "https://github.com/ManuelFay/colpali",
+		filter: false,
+		countDownloads: `path:"adapter_model.safetensors"`,
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:138]: Maybe let's do `adapter_config.json` for consistency with other adapter libraries

**Review by osanseviero — APPROVED** [review]
LGTM! It's ok to merge imo, as we still need to wait for a release and deploy here

**Comment by osanseviero** [comment]
(lint failing though)

File: packages/tasks/src/model-libraries.ts [hunk 3]
```diff
@@ -130,6 +130,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"asset/GPT.pt"`,
 	},
+	"colpali": {
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:133]:
```suggestion
	colpali: {
```

**Pull request merged**